src/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs

```diff
@@ -501,24 +522,14 @@ private static void ParseArgumentsIntoList(string arguments, List<string> result
                 // it should be added to the results and then reset for the next one.
                 if ((c == ' ' || c == '\t') && !inQuotes)
                 {
-                    if (currentArgument.Length > 0)
```

Comment:
Should lose the "and if we've built up any characters in the current argument," part of that comment.